app/pb_kits/playbook/plugins/pb_chart.js

```diff
@@ -1,9 +1,45 @@
 import Highcharts from 'highcharts'
 
 import { highchartsTheme } from '../pb_dashboard/pbChartsLightTheme.js'
+import colors from '../tokens/_colors.scss'
 
 require('highcharts/modules/variable-pie')(Highcharts)
 
+// Map Data Color String Props to our SCSS Variables
+const mapColors = (array) => {
+  const newArray = array.map((item) => {
```

Comment:
@jasperfurniss sorry for being late on this one, but do you think we can get the suggested refactoring below easily added to any existing PR? If not, I can open a new one.

```js
const mapColors = array => {
  const toSassKey = item => item.replace("-", "_")
  return array.map(item => colors[toSassKey(item)] || ""
}
```

Reply:
Yes. I'll add a new PR for playbook and test it locally. Thanks @drborges